service/src/main/java/bio/terra/drshub/util/AccessMethodUtils.java

```diff
@@ -0,0 +1,59 @@
+package bio.terra.drshub.util;
+
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+
+import bio.terra.drshub.config.DrsProvider;
+import bio.terra.drshub.generated.model.RequestObject.CloudPlatformEnum;
+import io.github.ga4gh.drs.model.AccessMethod;
+import io.github.ga4gh.drs.model.DrsObject;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
```

Comment:
I don't think this annotation is necessary at this time, since nothing is logged.